net.krafting.PleasureDVR.yml

```diff
@@ -0,0 +1,33 @@
+app-id: net.krafting.PleasureDVR
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: PleasureDVR.py
+finish-args:
+  -  --share=ipc
+  -  --share=network
+  -  --socket=fallback-x11
```

Comment:
Why only fallback-x11? Gtk4 supports native wayland.

Reply:
it was removed in the previous chanage, for I don't know why.